deployment/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.3'
+services:
+  manage:
+    build: ./scripts/mgmt
```

Comment:
I don't love the directory structure here. Since the Boto script requires a custom Dockerfile + requirements file + Python script in addition to `./deployment/scripts/manage`, I wanted to package them all up in a separate directory to avoid tossing a bunch of cruft directly in `./deployment/scripts`. I wound up going with `./deployment/scripts/mgmt/`, which is a sibling to `./deployment/scripts/manage`. Better ideas for directory structure are welcome!